chpl/chpl-resources/src/main/resources-dev/environment.properties

```diff
@@ -146,4 +146,10 @@ chplUrlBegin=https://chpl.ahrqdev.org
 
 #Asynchronous
 asyncListingDetailsEnabled=false
-asyncCacheRefreshEnabled=true
\ No newline at end of file
+asyncCacheRefreshEnabled=true
+
+#Website Change Request Email
+changeRequest.website.approval.subject=Website URL change request approved
+changeRequest.website.approval.body=Hello,<br/><br/>The website change request that was submitted on %s to change the website URL from %s to %s has been approved by %s.  The updated website URL will now be reflected in CHPL.<br/><br/>Thanks,<br/>The CHPL Team 
+changeRequest.website.pendingDeveloperAction.subject=Website URL change request requires further action
+changeRequest.website.pendingDeveloperAction.body=Hello,<br/><br/>The website change request that was submitted on %s to change the website URL from %s to %s requires further action.  %s has sent this change request back you for updating with the following comment:<br><ul><li>%s</li></ul><br/><br/>Thanks,<br/>The CHPL Team
```

Comment:
The "from %s to %s" part gives me pause. I noticed that the "from" value is being pulled from the Developer's current data, which means that if someone were to submit a CR, and a day or two later that Developer's website was updated some other way (upload, manual edit, whatever), then the email would have content that suggested the change was from the new data to the requested data, not the old data. I know it's an edge case, but if the text was just changed to read "change the website URL to %s" and just drop the old data then it wouldn't be an issue